server/test/suite/events.test.mjs

```diff
@@ -0,0 +1,130 @@
+"use strict";
+
+import { jest } from "@jest/globals";
+import * as events from "../../src/events.mjs";
+import { logger } from "../../src/logger.mjs";
+
+test("events.registerEventListener is working properly", () => {
+  const spy = jest.spyOn(logger, "debug");
+  const dumyObject = {
```

Comment:
"Dummy" has two Ms; if you're in the code making other changes, you can/should change, but it's obviously not high priority.

Reply:
done